src/PowerShellEditorServices/Services/TextDocument/Handlers/CompletionHandler.cs

```diff
@@ -184,7 +185,10 @@ public override async Task<CompletionItem> Handle(CompletionItem request, Cancel
                 // Retain PowerShell's sort order with the given index.
                 SortText = $"{sortIndex:D4}{result.ListItemText}",
                 FilterText = result.CompletionText,
-                TextEdit = textEdit // Used instead of InsertText.
+                // Used instead of Label when TextEdit is unsupported
+                InsertText = result.CompletionText,
```

Comment:
Ah so it just falls back then? That's handy!

How does `InsertText` work with things like type names where the completion text can be the whole type name? e.g. `[System.Management.Automation.<tab>`. Though even if that explodes it's better than not working at all so I'm cool with rollin' with it.

Reply:
> but also as I was tabbing through SMA it erroneously at some point doubled up the SM part

That's what I was expecting. My PR won't fix that, I'm even not 100% sure we *can* fix it if the client doesn't support edits. Either way I'm cool with it just to get those clients at least working and we can take a pass at figuring something out later